plasmapy/simulation/tests/test_particletracker.py

```diff
@@ -41,64 +31,79 @@ def sine(t, amplitude, omega, phase, mean):
     return params, stds
 
 
-# def test_particle_uniform_magnetic():
-#     r"""
-#         Tests the particle stepper for a uniform magnetic field motion.
-#     """
-#     test_plasma = uniform_magnetic_field()
+E_unit = u.V / u.m
```

Comment:
Precomputing this as per https://docs.astropy.org/en/stable/units/#performance-tips

Reply:
The performance bump was pretty horrible. It made interpolating fields the slowest part, actually.